### DISCUSSION TITLE
Dead?

### DISCUSSION THREAD

**Discussion by TechStudent10** [Avdan-OS/Website#352] [General]
As I've heard, the Discord server for this project is the better part of dead (no one noticed me joining back then leaving again). Maybe there should be a notice somewhere that notifies anyone following this project that it's most likely dead.

**Comment by Sammy99jsp** [comment]
Hmm.
The project's not really dead, though Website repo seems to be in maintenance-only mode— which is honestly fine, since it doesn't need to be changed dramatically.

**Reply by TechStudent10** [reply]
Sammy, it's been like a year since this post haha

Glad to see the project is still alive in some capacity

**Reply by Froxcey** [reply]
I can assure you, it is alive

<img src="https://github.com/Avdan-OS/Compositor/assets/51555391/d0379882-f2dc-42e1-962f-b3f122db656f" alt="vibe" width="60"/>
